deploying/deploying-to-heroku.md

```diff
@@ -38,33 +38,33 @@ Deployment is a fancy term for getting your website on the web. After building o
-- Read through this tutorial before we do it ourselves in the next section: https://devcenter.heroku.com/articles/getting-started-with-nodejs
+- Learn about deployment by going through the [Techtonica Slides on Deploying](https://docs.google.com/presentation/d/1Enwhd9hl1fn1-afMXJ6xvkJm5SDJpHjfQoA7s2znHpw/edit?usp=sharing.
+- Learn a little about Heroku by watching: [Video - What is Heroku (4 mins watch)](https://youtu.be/r5ZUQvl9BtE).
+- Go to https://devcenter.heroku.com/ and note the various languages Heroku supports.
+- Read through this tutorial before we do it ourselves in the next section: https://devcenter.heroku.com/articles/getting-started-with-nodejs.
 
 #### Deploying
 
 Now we'll work on deploying your app to Heroku.
 
 We'll be combining your frontend (create-react-app) with your backend (express) and deploying it to Heroku. Your frontend contains "static" JavaScript files -- when you deploy to Heroku, Heroku turns your whole React app into a couple of static files that it will serve to the browser. No matter what data you have in the database, these files will always be the same.
 
-Your backend, on the other hand, is dynamic -- when you make an API request, the backend runs javascript code to do things like reading and writing to a database. Unlike the React app, which always serves the same files to the browser, the backend will serve different information to the browser depending on what's in the database. We're going to combine your dynamic code (express), with your static code (create-react-app).
+Your backend, on the other hand, is dynamic -- when you make an API request, the backend runs javascript code to do things like reading and writing to a database. Unlike the React app, which always serves the same files to the browser, the backend will serve different information to the browser depending on what's in the database. We're going to combine your dynamic code (express) with your static code (create-react-app).
 
-1. cd into the React app you created and move _everything_ into a new directory named `client`:
+1. cd into the React app you created and move _everything_ into a new directory named `client`.
 
 ```
 cd <my react app>
 mkdir client
 mv * client
 ```
 
-2. Create a server directory. You will copy all the files from your Express API folder (1-3 JS files + package.json) into the `server` folder you're about to create inside your React app. _**This is where your API code will live from now on -- don't modify or use the old directory or repo**_
+2. Create a server directory. Copy all the files from your Express API folder (1-3 JS files + package.json) into the `server` folder you're about to create inside your React app. _**This is where your API code will live from now on -- don't modify or use the old directory or repo**_
 
 ```
 mkdir server
-cp my-express-server/* server
-# We need to keep package.json and node_modules at the top level.
+cp -r ~/path.to.source/. ~/path.to.destination/
```

Comment:
is this clear enough?  Can't they use the `server` directory they just created?

Reply:
Yes, clear enough.